packages/api/src/constants/assets.ts

```diff
@@ -0,0 +1,13 @@
+import { hashMessage } from 'fuels';
+
+export const ASSETS = {
+  FUEL_ETH: '0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07',
+};
+
+// create a hashMessage for fiat currencies
+// The keys are the currency codes, and the values are the asset IDs or other identifiers.
+export const FIAT_CURRENCIES = {
+  USD: hashMessage('USD'),
+  EUR: hashMessage('EUR'),
+  BRL: hashMessage('BRL'),
+};
```

Comment:
Prefer using hardcoded values in this case, it makes debugging and identifying them easier:
```suggestion
export const FIAT_CURRENCIES = {
  // hashMessage('USD')
  USD: '0x839...',
  // hashMessage('EUR')
  EUR: '0x123...',
  // hashMessage('BRL')
  BRL: '0x321...',
};
```